examples/defaultConfig.nix

```diff
@@ -16,6 +16,7 @@ let
 in
 {
   plugins = with pkgs.vimPlugins; with pkgs.vitalityVimPlugins;  [
+    { /**/ plugin = plenary-nvim; config = wrapLuaConfig (builtins.readFile ./config/lua-rocks-config.lua); }
```

Comment:
Lonely comment

Reply:
hehe yeah the comment is there to hijack the sort order 😂